qiskit/quantum_info/operators/operator.py

```diff
@@ -199,6 +199,49 @@ def from_label(cls, label):
                 op = op.compose(label_mats[char], qargs=[qubit])
         return op
 
+    @classmethod
+    def from_circuit(cls, circuit, use_layout=True, layout=None):
+        """Create a new Operator object from a :class`.QuantumCircuit`
+
+        While a :class:`.QuantumCircuit` object can passed directly as ``data``
+        to the class constructor
```

Comment:
I think a clause has gone missing here.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7616/commits/b6ebe91668be88faf76dca608a623d46a1770135